spec/routing/game_routing_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::GameController, type: :routing do
+  describe 'routing to the game endpoint /v1/game' do
```

Comment:
I appreciate the extra effort for testing, however routes are typically not tests. At least at the unit level. The integration/functional/acceptance tests are the ones that tend to cover the routes.

What does @cmvandrevala think?

Reply:
awaiting Cyrus's response